src/components/Confetti/Confetti.js

```diff
@@ -0,0 +1,12 @@
+const props = {
+  confettiParticles: {
+    type: Number,
+    default: 200,
+  },
+};
+
+const Confetti = {
```

Comment:
Please consider renaming this more obviously to: `ConfettiOverlay` and rename the files, stories, and directory as well.